src/Misc/SynthEngine.h

```diff
@@ -69,18 +69,20 @@ using std::unique_ptr;
 
 class SynthEngine
 {
-        const uint uniqueId;
-        Config Runtime;
-
-        GuiDataExchange::Connection<InterfaceAnchor> rootCon;
```

Comment:
...so my proposal would be _only_ to move rootCon down into a new `private:` section